js/src/forum/index.js

```diff
@@ -42,14 +47,24 @@ app.initializers.add('fof/oauth', () => {
             return orig(vnode);
         });
 
-        override(LogInButton.prototype, 'getButtonContent', function (orig) {
-            console.log(this.constructor.name);
-
-            return orig();
-        });
-
         extend(LogInButtons.prototype, 'view', function (vdom) {
             vdom.attrs.className += ' LogInButtons--icons';
         });
+
+        override(LogInModal.prototype, 'body', function () {
+            return [
+                <LogInButtons />,
+                (app.forum.attribute('fof-oauth').find(item => item && item.name === "google")
+                ? <div class="LogInButtons">
+                    <LogInButton className={'Button LogInButton--google'} icon={'fab fa-google'} path={'/auth/google'}>
```

Comment:
I feel like this log in button should be added along with the others. You can use the priority of an ItemList `items.add(name, content, priority)` to signify where it should go (higher priority appears later).

I don't like overriding LogInModal because that could cause unnecessary incompatibilities with other extensions. And it shouldn't be necessary for this scenario.